collagraph/collagraph.py

```diff
@@ -634,7 +634,11 @@ def key_to_event(key):
 
 
 def is_new(val, other, key):
-    return val != other.get(key)
+    other_value = other.get(key)
+    if other_value is None and val is not None:
```

Comment:
do you need to check this the other way around as well?

Reply:
I wanted to say 'no', but you actually have a good point! I'll update the test to also check for this.